public/index.html

```diff
@@ -7,6 +7,9 @@
     <script src="https://cdn.nimiq-testnet.com/web-offline.js"></script>
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
     <title>Nimiq Accounts Manager</title>
+
+    <link href="https://fonts.googleapis.com/css?family=Muli:400,600,700" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Fira+Mono&text=0123456789ABCDEFGHJKLMNPQRSTUVXY" rel="stylesheet">
```

Comment:
Ah, yes, we need a monospace. Has info on this been added to the style guide?

Reply:
Yes, `Fira Mono` is mentioned in the style guide.